server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/BlobManagerImpl.java

```diff
@@ -41,15 +41,15 @@
 
 import com.github.fge.lambdas.Throwing;
 
-public class StoreBlobManager implements BlobManager {
+public class BlobManagerImpl implements BlobManager {
```

Comment:
Good name!
Sometime, the old name makes me confuse

Reply:
To be fair I hesitated dropping the interface ;-)